.devcontainer/docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 services:
   devcontainer:
+    platform: linux/amd64
```

Comment:
interesting.

I had added this to the Dockerfile already: https://github.com/LabAutomationAndScreening/cloud-courier/blob/29645d8f01372b2b420dc98d153158405f7dddef/.devcontainer/Dockerfile#L4


You're saying you still got errors trying to run a devcontainer on your Mac?
What was the error?